docs/graveyard/README.md.old

```diff
@@ -0,0 +1,134 @@
+# GKE PoC Toolkit
```

Comment:
any reason we're keeping this file around? it will show up in the commit history

Reply:
Removed the graveyard folder. Good Call